charts/argo-cd/templates/argocd-server/ingress.yaml

```diff
@@ -9,7 +9,7 @@ metadata:
   labels:
     {{- include "argo-cd.labels" (dict "context" . "component" .Values.server.name "name" .Values.server.name) | nindent 4 }}
     {{- with .Values.server.ingress.labels }}
-      {{- toYaml . | nindent 4 }}
+      {{- tpl (toYaml .) . | nindent 4 }}
```

Comment:
`tpl (toYaml .) .` withing the `with` context will fail since `.` here refers to the current context (i.e. `.Values.server.ingress.labels`) and not the root context.

To refer to the root context, you should change the `tpl` call to `tpl (toYaml .) $`

Reply:
changed tpl call to tpl (toYaml .) $